components/Navbar/MobileNavbar/MobileNavbar.tsx

```diff
@@ -112,12 +120,19 @@ const MobileNavbar: React.FC<IProps> = () => {
           </div>
           <div className={'flex space-x-3 items-center'}>
             { connector &&
-              <div className="px-6 w-full py-2 border border-red-300 text-sm shadow-lg font-medium rounded-sm shadow-sm text-red-300 bg-gray-900 focus:outline-none ">
-                {tokenBalance.toFixed(3)} <span
-                  onClick={() => track()}
-                  className="hover:underline cursor-pointer">SSD</span>
-              </div>
+              <>
+                <div className="px-6 w-full py-2 border border-red-300 text-sm shadow-lg font-medium rounded-sm shadow-sm text-red-300 bg-gray-900 focus:outline-none ">
+                  {ssdTokenBalance.toFixed(3)} <span
+                    onClick={() => track()}
+                    className="hover:underline cursor-pointer">SSD</span>
+                </div>
+
+                <div className="px-6 w-full py-2 border border-red-300 text-sm shadow-lg font-medium rounded-sm shadow-sm text-red-300 bg-gray-900 whitespace-nowrap focus:outline-none ">
+                  {maticTokenBalance.toFixed(3)} <span>MATIC</span>
```

Comment:
did you decide that the naice add token to metamask with the track() function was overkill for matic since everything uses matic on polygon?

Reply:
oh yeah good question, MetaMask actually shows MATIC by default when you're connected to the Matic network.

<img width="234" alt="Screen Shot 2021-06-24 at 10 09 40 PM" src="https://user-images.githubusercontent.com/1393139/123363841-f6c62e00-d538-11eb-810d-be9739d656ce.png">

Reply:
native MATIC doesnt need to be tracked since it'll always be displayed in a wallet, WMATIC could have a track function but SSW does not use WMATIC

Reply:
welp, good thing y'all knew that since my metamask is ridiculous hahaha so it doesn't do that for me lulz